openfisca_core/indexed_enums.py

```diff
@@ -131,4 +131,4 @@ def __repr__(self):
         return '{}({})'.format(self.__class__.__name__, str(self.decode()))
 
     def __str__(self):
-        return str(self.decode_to_str())
+        return np.array2string(self.decode_to_str(),max_line_width=float("inf"))
```

Comment:
> Regular numpy arrays are wrapped when stringified. Why should EnumArray behave differently? I could see some issues when large population simulations.

Reply:
See #869 referenced in the PR description: we don't want arrays wrapped.

This is a design flaw in Numpy that this PR works around; arrays and strings are abstract data types, converting between array and string representation of the same underlying data should not be tied to a notion of "column width of the terminal" which is not only medium-specific but has no necessary relation to the terminal where the information will be viewed.

Admittedly this PR perhaps needs a unit test or two, I'm not actually sure the specific code for EnumArray is needed for the computation log case as they are a subclass of Numpy arrays.